### PR TITLE
0_2_3 release

### PR DESCRIPTION
**Pull request by sofroniewn**
# Description
Release notes for 0.2.3

### CODE REVIEW

File: docs/release/release_0_2_3.rst [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+API Changes
+-----------
+- None
+
+Bugfixes
+--------
+- Add numpydoc to default dependencies (#598)
+- fix dimensions change when no layers (#603)
+- fix bracket highlighting (#606)
+- fix for data array overwriting during 3D rendering (#613)
```
Comment (jni) [inline docs/release/release_0_2_3.rst]: can you specify that this applies only to float32? We don't want people to get too alarmed. =P

File: docs/release/release_0_2_3.rst [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+Deprecations
+------------
+- None
+
+4 authors added to this release [alphabetical by first name or login]
+---------------------------------------------------------------------
+- Juan Nunez-Iglesias
+- Nicholas Sofroniew
+- Talley Lambert
+- wconnell (wconnell)
```
Comment (jni) [inline docs/release/release_0_2_3.rst]: Update to Will Connell?

**Review by jni — APPROVED** [review]
These look great! I've left just two minor comments.

File: docs/release/release_0_2_3.rst (hunk 2 above)
Comment (sofroniewn) [reply docs/release/release_0_2_3.rst]: done

File: docs/release/release_0_2_3.rst (hunk 1 above)
Comment (sofroniewn) [reply docs/release/release_0_2_3.rst]: done

**Pull request merged**